backend/devices/bitbox02/device.go

```diff
@@ -338,9 +339,17 @@ func (device *Device) CreateBackup() error {
 	if device.status != StatusSeeded {
 		return errp.New("invalid status")
 	}
+
+	now := time.Now()
+	timestamp := now.Unix()
```

Comment:
nit: same comment as before, no need for a var that is used only once. feel free to keep or change ;)